stdlib/log-api/src/main/java/org/ballerinalang/stdlib/log/AbstractLogFunction.java

```diff
@@ -77,7 +75,15 @@ public String get() {
                 return msg;
             }
         };
-        if (LOG_MANAGER.getPackageLogLevel(pckg).value() <= logLevel.value()) {
+        String pckg = ".";
+        BLogLevel enabledLogLevel;
+        if (LOG_MANAGER.isModuleLogLevelEnabled()) {
+            pckg = getPackagePath();
+            enabledLogLevel = LOG_MANAGER.getPackageLogLevel(pckg);
+        } else {
+            enabledLogLevel = LOG_MANAGER.getGlobalLogLevel();
+        }
+        if (enabledLogLevel.value() <= logLevel.value()) {
```

Comment:
Doesn't this logic give the wrong module name in the log when a module specific log level is not set? For example, consider there's a log call in a function in a module `foo` like `log:printInfo("Hello World!");` and this module is used in my project's module `bar`. The printed log should look like something like,
```
2020-05-27 15:30:27,539 INFO  [pubudu/foo] - Hello World! 
```
But with this change, it would print it like,
```
2020-05-27 15:30:27,539 INFO  [] - Hello World! 
```

Reply:
The module name printed with the log should always be the point of origin of that particular log call.